src/devices/Mpr121/Mpr121.cs

```diff
@@ -0,0 +1,252 @@
+                }
+
+                if (isStatusChanged)
+                {
+                    OnPropertyChanged(nameof(ChannelStatuses));
+                }
+            }
+        }
+
+        /// <summary>
+        /// Gets ot sets the period in milliseconds to refresh the channels statuses.
+        /// </summary>
+        /// <remark>
+        /// Set value 0 to stop the automatically refreshing. Setting the value greater than 0 will start/update auto-refresh.
+        /// </remark>
+        public int PeriodRefresh
+        {
+            get { return _periodRefresh; }
+
+            set
+            {
+                _periodRefresh = value;
+
+                if (_periodRefresh > 0 && _timer == null)
+                {
+                    _timer = new Timer(RefreshChannelStatuses, this, TimeSpan.FromMilliseconds(_periodRefresh), TimeSpan.FromMilliseconds(_periodRefresh));
+                }
+                else if (_periodRefresh == 0 && _timer != null)
+                {
+                    _timer.Dispose();
+                    _timer = null;
+                }
+                else if (_periodRefresh > 0 && _timer != null)
+                {
+                    _timer.Change(TimeSpan.FromMilliseconds(_periodRefresh), TimeSpan.FromMilliseconds(_periodRefresh));
+                }
+            }
+        }
+
+        /// <summary>
+        /// Initialize a MPR121 controller with default configuration. Auto-refreshing of channel statuses is disabled.
+        /// </summary>
+        /// <param name="device">The i2c device.</param>
+        public Mpr121(I2cDevice device) : this(device, 0, GetDefaultConfiguration())
+        {
+        }
+
+        /// <summary>
+        /// Initialize a MPR121 controller with default configuration.
+        /// </summary>
+        /// <param name="device">The i2c device.</param>
+        /// <param name="periodRefresh">The period in milliseconds of refresing the channel statuses.</param>
+        public Mpr121(I2cDevice device, int periodRefresh) : this(device, periodRefresh, GetDefaultConfiguration())
+        {
+        }
+
+        /// <summary>
+        /// Initialize a MPR121 controller with custom configuration. Auto-refreshing of channel statuses is disabled.
+        /// </summary>
+        /// <param name="device">The i2c device.</param>
+        /// <param name="configuration">The controller configuration.</param>
+        public Mpr121(I2cDevice device, Mpr121Configuration configuration) : this(device, 0, configuration)
+        {
+        }
+
+        /// <summary>
+        /// Initialize a MPR121 controller.
+        /// </summary>
+        /// <param name="device">The i2c device.</param>
+        /// <param name="periodRefresh">The period in milliseconds of refresing the channel statuses.</param>
+        /// <param name="configuration">The controller configuration.</param>
+        public Mpr121(I2cDevice device, int periodRefresh, Mpr121Configuration configuration)
```

Comment:
I'd leave just this constructor with default values. I.e.:

```csharp
public Mpr121(I2cDevice device, int periodRefresh = 0, Mpr121Configuration configuration = null)
{
    configuration = configuration ?? GetDefaultConfiguration();
...
```

Reply:
Fixed.